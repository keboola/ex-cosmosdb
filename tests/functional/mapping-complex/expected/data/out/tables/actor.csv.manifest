{
    "incremental": false,
    "columns": [
        "name",
        "movie_pk"
    ],
    "column_metadata": {
        "name": [
            {
                "key": "KBC.datatype.nullable",
                "value": true
            },
            {
                "key": "KBC.datatype.basetype",
                "value": "string"
            }
        ],
        "movie_pk": [
            {
                "key": "KBC.datatype.nullable",
                "value": true
            },
            {
                "key": "KBC.datatype.basetype",
                "value": "string"
            }
        ]
    }
}