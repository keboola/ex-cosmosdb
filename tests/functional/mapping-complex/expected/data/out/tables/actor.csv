"Daniel Bruhl","Rush"
"Chris Hemsworth","Rush"
"Olivia Wilde","Rush"
"Hugh Jackman","Prisoners"
"Jake Gyllenhaal","Prisoners"
"Viola Davis","Prisoners"
"Patrick Wilson","Insidious: Chapter 2"
"Rose Byrne","Insidious: Chapter 2"
"Barbara Hershey","Insidious: Chapter 2"
